### PR TITLE
Fix : Agenda section shows blank if user language is French [CPNHUB-231]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: assets/components/CalendarButton.jsx [hunk 1]
```diff
@@ -28,8 +26,12 @@ class CalendarButton extends React.Component {
     render() {
         const isStartDateToday = moment.isMoment(this.state.startDate) && !this.state.startDate.isSame(moment(), 'day');
         const datePicker = (<DatePicker
-            customInput={<CalendarButtonWrapper active={isStartDateToday}/>}
-            dateFormat={AGENDA_DATE_PICKER_FORMAT_SHORT}
+            customInput={
+                <CalendarButtonWrapper
+                    active={isStartDateToday}
+                    date={moment(this.state.startDate).format('YYYY-MM-DD')}
```
Comment (petrjasek) [inline assets/components/CalendarButton.jsx:32]: I don't see date prop in `CalendarButtonWrapper` so not sure what this does actually?

Comment (devketanpro) [reply assets/components/CalendarButton.jsx:32]: my bad its typo. here we have to pass `value` prop

**Comment by petrjasek** [comment]
what about that default config?

**Comment by devketanpro** [comment]
with these changes 

for fr :- 
image
and for en :-
image

**Pull request merged**